tsconfig.json

```diff
@@ -8,10 +8,10 @@
     "reactNamespace": "React",
     "module": "ESNext",
     "moduleResolution": "Node",
+    "sourceMap": true,
     "paths": {
-      "~/*": [
-        "/*"
-      ]
+      "~/*": ["/*"],
+      "@divriots/starter-react-aria/*": ["*"]
```

Comment:
`*` works as well ? or should be `/*` ?

Reply:
Yeah * works as well: http://localhost:5000/edit/eFLUQfY0zrR6dOuX5llT/light/doc/doc.scss?branch=~main@3SozDfydSvf9z3UAsAat8O9zQ3B2